.github/CODEOWNERS

```
@@ -559,12 +559,14 @@
 /core/templates/services/autoplayed-videos.service*.ts @aks681
 /core/templates/services/external-save.service*.ts @srijanreddy98
 /core/templates/services/external-rte-save.service*.ts @srijanreddy98
+/core/templates/services/oppia-rte-parser.service.ts @srijanreddy98 @aks681
```

```suggestion
/core/templates/services/oppia-rte-parser.service*.ts @srijanreddy98 @aks681
```

Are we expecting this to catch the test files as well?

---

Yes, we are. Changed it.